python/lsst/pipe/tasks/finalizeCharacterization.py

```diff
@@ -169,7 +169,23 @@ def setDefaults(self):
                                      'slot_Centroid_flag',
                                      'base_GaussianFlux_flag']
 
-        self.measure_ap_corr.sourceSelector['flagged'].field = 'calib_psf_used'
+        # Configure aperture correction to select only high s/n sources (that
+        # were used in the psf modeling) to avoid background problems when
+        # computing the aperture correction map.
+        self.measure_ap_corr.sourceSelector = 'science'
```

Comment:
Is this missing a `.name`?

Reply:
`pex_config` 🪄 , it works either way.

Reply:
Ah...this line followed by the one below looks very odd to the muggles 😉